### DISCUSSION TITLE
Improve refresh warning messages in SDK

### DISCUSSION THREAD

**Discussion by MrCasquette** [directus/directus#14277] [Feature Requests]
### Describe the Bug

When i login, then logout and try to create a user I got this message. When i try a second time the error is gone... Not so User friendly even it's a situation that no one will probably go through (but i DID ^^), I think something went wrong and could probably exist with a more usual situation. Thank you for the help

### To Reproduce

Use the SDK, login, logout and sign up (create a user)

### Errors Shown

refresh_token is required in either the json payload or cookie

### What version of Directus are you using?

9.13

### What version of Node.js are you using?

v12.22.5

### What database are you using?

Mysql

### What browser are you using?

Chrome

### How are you deploying Directus?

Self-Hosted on Ubuntu

**Comment by br41nslug** [comment]
are you by any chance using next.js on the frontend like in https://github.com/directus/directus/discussions/13825 ?

**Reply by MrCasquette** [reply]
You're Right. I put an answer on the ticket

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
